Makefile

```diff
@@ -42,7 +42,13 @@ test:
 	python tests/tlstest.py server localhost:4433 tests & sleep 4
 	python tests/tlstest.py client localhost:4433 tests
 
-test-local:
```

Comment:
To avoid potential clash with unstaged files, you should have also the
```
.PHONY : test-utils
```
some place in the Makefile. I also see missing `test-local test test-dev tacks dist` targets in the `.PHONY`, which is out of the scope of this specific change, but would make sense to fix at some point.